apps/backend/backend-main/src/modules/order/buyFromAStore/buyFromAStore.service.ts

```diff
@@ -240,23 +240,26 @@ export class BuyFromStoreService {
       throw new BuyFromStoreNotFoundError(`Order ID ${orderId} not found`);
     }
 
-    if (order.status !== OrderStatusEnum.COMPLETED) {
-      throw new BuyFromStoreOrderNotCompletedError(
-        `Cannot review an incomplete order with ID ${orderId}`,
-      );
-    }
+    // if (order.status !== OrderStatusEnum.COMPLETED) {
+    //   throw new BuyFromStoreOrderNotCompletedError(
+    //     `Cannot review an incomplete order with ID ${orderId}`,
+    //   );
+    // }
```

Comment:
I don't think this should be removed, any reason why this validation was removed?

```suggestion
     if (order.status !== OrderStatusEnum.COMPLETED) {
       throw new BuyFromStoreOrderNotCompletedError(
         `Cannot review an incomplete order with ID ${orderId}`,
       );
    }
```

Reply:
For testing purpose, once pushing it `main` branch these validations will be added

Reply:
We dont need to run migrations. Check slack